tests/test_xlsx2json.py

```diff
@@ -73,6 +78,78 @@ def test_conversion_2_json(self) -> None:
         resolver.store['eva-biosamples.json'] = biosample_json_schema
         jsonschema.validate(json_data, eva_json_schema, resolver=resolver)
 
+
+    def test_conversion_2_json_V2(self) -> None:
+        xls_filename = os.path.join(self.resource_dir, 'EVA_Submission_test_V2.xlsx')
+        self.parser = XlsxParser(xls_filename, self.conf_filename_v2)
+        output_json = os.path.join(self.resource_dir, 'EVA_Submission_test_output_v2.json')
+        errors_yaml = os.path.join(self.resource_dir, 'EVA_Submission_test_errors_v2.yml')
+        self.parser.json(output_json)
+        self.parser.save_errors(errors_yaml)
+
+        # confirm no errors
+        with open(errors_yaml) as open_file:
+            errors_data = yaml.safe_load(open_file)
+            assert errors_data == []
+
+        with open(output_json) as open_file:
+            json_data = json.load(open_file)
+            # assert json file is created with expected data
+            assert sorted(json_data.keys()) == ['$schema', 'analysis', 'files', 'project', 'sample', 'submitterDetails']
+            json_data.pop('$schema', None)
+            self.assertEqual(self.get_expected_json(), json_data)
+
+        # assert json schema
+        with open(self.eva_schema) as eva_schema_file:
+            eva_json_schema = json.load(eva_schema_file)
+        with open(self.biosample_schema) as biosample_schema_file:
+            biosample_json_schema = json.load(biosample_schema_file)
+
+        # assert created json file sample field conforms to eva-biosamples schema
+        jsonschema.validate(json_data['sample'][3]['bioSampleObject'], biosample_json_schema)
```

Comment:
We've [removed usage of the eva-biosample schema](https://github.com/EBIvariation/eva-sub-cli/pull/120), so once you rebase you should be able to simplify these tests.